exporters/memory/include/opentelemetry/exporters/memory/in_memory_span_exporter.h

```diff
@@ -65,6 +65,13 @@ class InMemorySpanExporter final : public opentelemetry::sdk::trace::SpanExporte
     return sdk::common::ExportResult::kSuccess;
   }
 
+  /**
+   * Force flush the exporter.
+   * @param timeout an option timeout, default to max.
+   * @return return true when all data are exported, and false when timeout
+   */
+  bool ForceFlush(std::chrono::microseconds /* timeout */) noexcept override { return true; }
```

Comment:
Sorry to be picky here, but should we let sdk::Exporter::ForceFlush() method be called for all exporters with default implementation ?

Reply:
Why do we need call `sdk::Exporter::ForceFlush()` ?  The default implementation in `sdk::Exporter` is just for the old code without thde default implementation, it just do nothing.

Reply:
I meant we don't need this method for the exporters which don't have anything to flush, let the base class method to get called by default.

Reply:
Thanks, it's removed now.